skops/card/_model_card.py

```diff
@@ -1155,6 +1155,46 @@ def add_permutation_importances(
 
         return self
 
+    def add_fairlearn_metric_frame(
+        self,
+        metric_frame,
+        table_name: str = "Fairlearn MetricFrame Table",
+        transpose=True,
+    ) -> Card:
```

Comment:
should we be adding a `description` to our `add_*` methods? From the user's perspective, it seems odd that it's tricky to add a description here for sections. I think it'd make sense for all of them to allow a description as well as a title. WDYT? also cc @skops-dev/maintainers

Reply:
Yes, I agree it would be good to have, some existing methods also don't have it, e.g. `add_table`. It's probably easier to have a separate PR where this argument is added.

Reply:
@adrinjalali For clarification, should I add the `description` argument to this method in this PR? Or do it in another one?

Reply:
we can keep it for another PR as @BenjaminBossan suggested.

Reply:
One more thing: Could you please change the type annotation to return `-> Self`? This is a change we recently made on all the other methods as well.